static/app/routes.tsx

```diff
@@ -1728,11 +1728,6 @@ function buildRoutes() {
           )}
         />
       </Route>
-      <Route path={`${MODULE_BASE_URLS[ModuleName.UPTIME]}/`}>
```

Comment:
I don't think you'll be able to remove these routes completely until the new nav is fully GA'd. You'll need to make sure it still works in the old nav (you can test by toggling it on/off in the help menu).

You can use `usePrefersStackedNav()` to check which whether or not the new nav is active

Reply:
ah right 😢